modules/nw-ovn-ipsec-verification.adoc

```diff
@@ -10,30 +10,29 @@ As a cluster administrator, you can verify that IPsec is enabled.
 
 .Verification
 
-. To find the names of the OVN-Kubernetes control plane pods, enter the following command:
+. To find the names of the OVN-Kubernetes data plane pods, enter the following command:
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-master
+$ oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-node
 ----
 +
 .Example output
 [source,terminal]
 ----
-ovnkube-master-4496s        1/1     Running   0          6h39m
-ovnkube-master-d6cht        1/1     Running   0          6h42m
-ovnkube-master-skblc        1/1     Running   0          6h51m
-ovnkube-master-vf8rf        1/1     Running   0          6h51m
-ovnkube-master-w7hjr        1/1     Running   0          6h51m
-ovnkube-master-zsk7x        1/1     Running   0          6h42m
+ovnkube-node-5xqbf                       8/8     Running   0              28m
+ovnkube-node-6mwcx                       8/8     Running   0              29m
+ovnkube-node-ck5fr                       8/8     Running   0              31m
+ovnkube-node-fr4ld                       8/8     Running   0              26m
+ovnkube-node-wgs4l                       8/8     Running   0              33m
+ovnkube-node-zfvcl                       8/8     Running   0              34m
 ----
 
 . Verify that IPsec is enabled on your cluster:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> \
-  ovn-nbctl --no-leader-only get nb_global . ipsec
+$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-node-<XXXXX> ovn-nbctl --no-leader-only get nb_global . ipsec
```

Comment:
@sjhala-ccs I'm not sure I follow. The verification step is part of `modules/nw-ovn-ipsec-verification`